### PR TITLE
Add code of conduct doc

### PR DESCRIPTION
**Pull request by Abdiramen**

### CODE REVIEW

**Comment by Abdiramen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #45** <a href="https://app.graphite.dev/github/pr/ngrok/ngrok-python/45" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ngrok/ngrok-python/45?utm_source=stack-comment).

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# ngrok Documentation Code of Conduct
```
Comment (bobzilladev) [inline CODE_OF_CONDUCT.md:1]: Remove `Documentation` here :)

Comment (Abdiramen) [reply CODE_OF_CONDUCT.md:1]: woops

Comment (CK-Ward) [reply CODE_OF_CONDUCT.md:1]: plus one to that

Comment (Abdiramen) [reply CODE_OF_CONDUCT.md]: fixed here: https://github.com/ngrok/ngrok-python/pull/45/commits/312762e6a6c57cefbc4dae24898531de6ecef8bd

**Review by jrobsonchase — APPROVED** [review]

**Review by bobzilladev — APPROVED** [review]

**Pull request merged**